datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx

```diff
@@ -24,13 +25,26 @@ const HelpIcon = styled(QuestionCircleOutlined)`
 
 type Props = {
     rowCount?: number | null;
+    columnCount?: number | null;
+    sizeInBytes?: number | null;
     queryCountLast30Days?: number | null;
     uniqueUserCountLast30Days?: number | null;
     lastUpdatedMs?: number | null;
 };
 
+const formatBytesStat = (bytes: number) => {
+    const formattedBytes = formatBytes(bytes);
+    return (
+        <Tooltip title={`This dataset consumes ${formatNumberWithoutAbbreviation(bytes)} bytes of storage.`}>
+            <b>{formattedBytes.number}</b> {formattedBytes.unit}
+        </Tooltip>
+    );
+};
```

Comment:
this is really more of a new component and not so much a helper function. so maybe we call this `FormattedBytesStat` that accepts `bytes` as a prop?

Reply:
yeah i like that!